.envrc

```diff
@@ -0,0 +1 @@
+dotenv
```

Comment:
`direnv` looks for `.envrc` rather than `.env`. However, `direnv` has the ability to load what's in `.env` by specifying this `dotenv` line in this file.